### DISCUSSION TITLE
Fresh mount on Rewards Catalogue

### DISCUSSION THREAD

**Discussion by SignorSimone** [chromiecraft/chromiecraft#7048] [Suggestions]
Add new items and mounts that can be purchased with chromie points.

**Comment by heyitsbench** [comment]
Harder than you'd think, there's only so many mounts that exist in the game files to be added.

**Comment by SignorSimone** [comment]
do you plan to add Reins of the Bengal Tiger and Horn of the Red Wolf?

**Comment by ChaoticMe** [comment]
I just want to throw money at the ZG Tiger. None of the kitties on the shop can be used by Horde, except the spectral. If I could buy any of the other cats for my Horde mains, that'd be a great solution too.

**Reply by SignorSimone** [reply]
Me too
